### ISSUE TITLE
ValueError: Columns must be same length as key

### ISSUE THREAD

**Issue by philsv** [git-shogg/finsec#9]
@git-shogg 

running the following:

```
cik = '0001811907'
filing = Filing(cik, declared_user=f"{os.getenv('DECLARED_USER')}")
filing_df = filing.latest_13f_filing()
```

gave me a

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/philsv/.local/lib/python3.11/site-packages/finsec/filing.py", line 15, in latest_13f_filing
    return self.get_latest_13f_filing(simplified, amend_filing)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/philsv/.local/lib/python3.11/site-packages/finsec/base.py", line 250, in get_latest_13f_filing
    latest_13f_cover_page, latest_holdings_table, latest_simplified_holdings_table = self._apply_amendments(qtr_year_str, latest_13f_cover_page, latest_holdings_table, latest_simplified_holdings_table)
                                                                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/philsv/.local/lib/python3.11/site-packages/finsec/base.py", line 198, in _apply_amendments
    self._13f_amendment_filings_period_of_filings()
  File "/home/philsv/.local/lib/python3.11/site-packages/finsec/base.py", line 71, in _13f_amendment_filings_period_of_filings
    self._13f_amendment_filings[['Period of Report','Period of Report Quarter Year']]  = self._13f_amendment_filings.apply(_pandas_apply_func,axis=1)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/philsv/.local/lib/python3.11/site-packages/pandas/core/frame.py", line 4287, in __setitem__
    self._setitem_array(key, value)
  File "/home/philsv/.local/lib/python3.11/site-packages/pandas/core/frame.py", line 4329, in _setitem_array
    check_key_length(self.columns, key, value)
  File "/home/philsv/.local/lib/python3.11/site-packages/pandas/core/indexers/utils.py", line 390, in check_key_length
    raise ValueError("Columns must be same length as key")
ValueError: Columns must be same length as key
```

Could you check on your side?

**Comment by git-shogg** [comment]
Hey @philsv, thanks for flagging this. It was a bug with the package, the package was looking for amendment filings where there were none for Citadel. This has been resolved with version 0.0.12 (available via `pip install finsec`).

**Comment by philsv** [comment]
Thank you @git-shogg
